docs/pages/lightclient-prover/light-client.md

```diff
@@ -0,0 +1,108 @@
+
+## Light-Client CLI Example
+
+It is possible to start up the light-client as a standalone process.
+
+```bash
+lodestar lightclient \
+    --network mainnet \
+    --beacon-api-url https://beacon-node.your-domain.com \
+    --checkpoint-root "0xccaff4b99986a7b05e06738f1828a32e40799b277fd9f9ff069be55341fe0229"
+```
+
+## Light-Client Programmatic Example
+
+For this example we will assume there is a running beacon node at `https://beacon-node.your-domain.com`
+
+```ts
+import {Api} from "@lodestar/api/beacon";
+import {ApiError} from "@lodestar/api";
+import {Bytes32} from "@lodestar/types";
+import {createChainForkConfig} from "@lodestar/config";
+import {networksChainConfig} from "@lodestar/config/networks";
+import {
+    GenesisData,
+    Lightclient,
+    LightclientEvent,
+    RunStatusCode,
+    getLcLoggerConsole
+} from `@lodestar/lightclient`;
+
+async function getGenesisData(api: Pick<Api, "beacon">): Promise<GenesisData> {
+    const res = await api.beacon.getGenesis();
+    ApiError.assert(res);
+
+    return {
+        genesisTime: Number(res.response.data.genesisTime),
+        genesisValidatorsRoot: res.response.data.genesisValidatorsRoot,
+    };
+}
+
+async function getSyncCheckpoint(api: Pick<Api, "beacon">): Promise<Bytes32> {
+    const res = await api.beacon.getStateFinalityCheckpoints("head");
+    ApiError.assert(res);
+    return res.response.data.finalized.root;
+}
+
+const config = createChainForkConfig(networksChainConfig.mainnet);
+
+const logger = getLcLoggerConsole({logDebug: Boolean(process.env.DEBUG)});
+
+const api = getClient({urls: ["https://beacon-node.your-domain.com"]}, {config});
+
+const transport = new LightClientRestTransport(api);
+
+const lightclient = await Lightclient.initializeFromCheckpointRoot({
+    config,
+    logger,
+    transport,
+    genesisData: await getGenesisData(api),
+    checkpointRoot: await getSyncCheckpoint(api),
+    opts: {
+        allowForcedUpdates: true,
+        updateHeadersOnForcedUpdate: true,
+    }
+});
+
+// Wait for the lightclient to start
+await new Promise<void>((resolve) => {
+    const lightclientStarted = (status: RunStatusCode): void => {
+        if (status === RunStatusCode.started) {
+            this.lightclient?.emitter.off(LightclientEvent.statusChange, lightclientStarted);
+            resolve();
+        }
+    };
+    lightclient?.emitter.on(LightclientEvent.statusChange, lightclientStarted);
+    logger.info("Initiating lightclient");
+    lightclient?.start();
+});
+
+logger.info("Lightclient synced");
+
+lightclient.emitter.on(LightclientEvent.lightClientFinalityHeader, async (finalityUpdate) => {
+    console.log(finalityUpdate);
+});
+
+lightclient.emitter.on(LightclientEvent.lightClientOptimisticHeader, async (optimisticUpdate) => {
+    console.log(optimisticUpdate);
+});
+```
```

Comment:
Code usage examples should live in the package README or examples folder and be ensured to run against the current version. 

We run check-readme for that https://github.com/ChainSafe/lodestar/blob/8afbc98b561d714fdadd1946ec2a5aab83f479de/package.json#L23

Reply:
sweet!  updated.  I moved that section (and the rest from that file) to `packages/light-client/README.md` and then moved that whole file into the docs from `prepare-docs.sh`.  I did the same for the `prover` so that that README also gets included into the main docs.